src/Platform.h

```diff
@@ -82,7 +82,7 @@ enum ConfigEntry
     Firm_Message,
     Firm_MAC,
 
-    AudioBitrate,
+    AudioBitrate
```

Comment:
Why has the `,` been removed here?

Reply:
I had previously exposed the Discord related settings there thinking that's just where settings belong, until I realized they don't, so I removed them again and that comma went away in the process